docs/content/en/docs/reference/tetragon-configuration.md

```diff
@@ -0,0 +1,197 @@
+---
+title: "Tetragon Daemon Configuration"
+linkTitle: "Tetragon Daemon Configuration"
```

Comment:
```suggestion
title: "Agent configuration"
```
Maybe? most importantly we don't need to say Tetragon again here. And agent is maybe more common than daemon.

Reply:
Thanks, both seem fine.Tetragon to differentiate from tetra